fmriprep/utils/bids.py

```diff
@@ -195,8 +195,9 @@ def validate_input_dir(exec_env, bids_dir, participant_label):
             "MALFORMED_BVAL",
             "MISSING_TSV_COLUMN_EEG_ELECTRODES",
             "MISSING_SESSION",
+            "NO_T1W",
```

Comment:
We shouldn't ignore this, in any case. It can be a warning or an error.

```suggestion
```

Reply:
Oh my bad, I mistakenly read this array was for warning not ignores. 🤦🏼 reverting that.